docs/routers/nokia-G240WB.md

```diff
@@ -0,0 +1,112 @@
+This guide was developed using a `Nokia G-240W-B` router. It should work for other models in the `G` series.
+
+### Device information
+
+- Device name: `Nokia G-240W-B`
+- Vendor: `Nokia`
+- Hardware version: `3FE56756ADBA`
+- Software version: `3FE47969AGCA21`
+- Chipset: `BCM6838`
+
+> Note:
+There is no single way to do it right. Chose the one best suited for you.
+This guide is `IPv4` and `IPv6` compatible, but it is not mandatory to use `IPv6`.
+
+### Prerequisites
+
+- The `Raspberry Pi` is accessible from the `LAN` network via `SSH`.
+- The `Raspberry Pi` is using static IP address, `IPv4` and `IPv6` as well.
+- The `ISP` provides `IPv6` address, prefix and gateway to the router.
+- `Pi-hole` has enabled `DHCP server`.
```

Comment:
Why do you need Pi-hole to be the DHCP server?

Reply:
Good question, actually this is not necessary since the Router can serve as DHCP server by default.
Removed from guide.